.github/workflows/create-release.yml

```diff
@@ -26,55 +26,78 @@ env:
   RUN_RECEIPT_FILENAME: "run-receipt.cyclonedx.json"
   PATCHED_USNS_FILENAME: "patched-usns.json"
 jobs:
+  get_architectures:
+    name: Get Architectures
+    runs-on: ubuntu-22.04
+    outputs:
+      architectures: ${{ steps.lookup.outputs.platforms }}
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Lookup Supported Architectures
+      id: lookup
+      run: |
```

Comment:
Can we add the usual boiler plate here please?

```
#! /usr/bin/env bash

set -euo pipefail
shopt -s inherit_errexit
```

I can forsee a failure to download `yj` being confusing to debug :)

Reply:
addressed in https://github.com/paketo-buildpacks/jammy-tiny-stack/compare/10213882af6cff681575c20de56649627a552a97..18bd7f895d73412ff2735dfc388f1084dcf9cd33